Debug App/Tests/Unit Tests/Data Models/IPay88Tests.swift

```diff
@@ -114,26 +115,27 @@ class IPay88Tests: XCTestCase {
         
         AppState.current.apiConfiguration = PrimerAPIConfiguration(
             coreUrl: decodedClientToken.coreUrl,
-            pciUrl: decodedClientToken.pciUrl,
+            pciUrl: decodedClientToken.pciUrl, 
+            assetsUrl: "https://assets.staging.core.primer.io",
             clientSession: ClientSession.APIResponse(
                 clientSessionId: "client-session-id",
                 paymentMethod: nil,
                 order: nil,
                 customer: nil,
                 testId: nil),
             paymentMethods: [
-
+                
```

Comment:
There are a lot of diffs like this, some adding spaces, some removing. Maybe something is up with Swiftlint?

Reply:
I think SwftLint is fixing trailing_whitespace violations.